dysnix/geth/templates/scripts/_s3-env.tpl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+export S5CMD=/s5cmd
+
+# chaindata options
+export EXCLUDE_ANCIENT="--exclude *.cidx --exclude *.ridx --exclude *.cdat --exclude *.rdat"
+export EXCLUDE_CHAINDATA="--exclude *.ldb --exclude *.sst"
```

Comment:
Does it support "old" and "new" geth databases out of the box ?

Reply:
Yes, it must work, added ".sst" file extension which is for `pebble` state backend.